Src/ReleaseHistory.md

```diff
@@ -13,6 +13,7 @@
 - Rename `SEC101/050.IdentifiableNpmLegacyAuthorToken` to `SEC101/050.NpmIdentifiableAuthorToken` [#683](https://github.com/microsoft/sarif-pattern-matcher/pull/683)
 - Rename `SEC101/017.NpmAuthorToken` to `SEC101/017.NpmAuthorToken` [#683](https://github.com/microsoft/sarif-pattern-matcher/pull/683)
 - Rename `SEC101/006.GitHubPat` to `SEC101/006.GitHubLegacyPat` [#678](https://github.com/microsoft/sarif-pattern-matcher/pull/678)
+- Disable `SEC101/029.AlibabaCloudCredentials` [#700](https://github.com/microsoft/sarif-pattern-matcher/pull/700)
```

Comment:
>AlibabaCloudCredentials [](http://example.com/codeflow?start=22&length=23)

Provide an explanation of why? Typically we would reference the exception that this rule is raising. Are you going to bring this check back? Think of it from the user standpoint.

Disable `SEC101/029.Alibaba` as it raises `XX` exceptions during validation producing messages such as `XXX`. These exceptions are caused by incompatibilities between the Alibaba supporting code and .net core 3.1 and 6.0. We should investigate restoring the rule when these dependencies are updated.